### PR TITLE
Created the new webpage and uses the tinygrad webpage as reference for it

### PR DESCRIPTION
**Pull request by SyedAddan**
Created a new webpage and added/changed these things
1. Overview section
2. Upadates section
3. Contributing section
4. FAQ section

### CODE REVIEW

**Comment by fakhirali** [comment]
I think remove the update section for now. 
Make some space for demo videos/audios.
Installation/get started section that shows the easiest way to install and start using it.

Bounty Locked! Thanks :lock:

**Comment by fakhirali** [comment]
Also do you have a color scheme in mind? Is it possible to incorporate some colors here?

File: index.html [hunk 1]
```diff
@@ -1,53 +1,192 @@
...
+          the supported models guide
+        </a>
+        to learn about the models currently supported in OVC.
+      </dd>
+      <dt>Q. How does it work?</dt>
+      <dd>
+        A. Look at
+        <a href="https://github.com/fakhirali/OpenVoiceChat">the code on github</a>
+        or 
+        <a href="https://github.com/fakhirali/OpenVoiceChat/blob/webpage-overhaul/notes/Models.md">read here</a>.
```
Comment (fakhirali) [inline index.html:188]: This should point to working.md

**Comment by SyedAddan** [comment]
On it. I guess we should change the branches as well. From webpage-overahaul to master?

**Comment by fakhirali** [comment]
No no abhi nahi. Main merge kar doon ga

**Comment by SyedAddan** [comment]
No I meant the links. Links me branches webpage-overhaul waali aarahi hain

**Comment by SyedAddan** [comment]
Yes, already done

**Pull request merged**